WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -28,12 +28,12 @@
             <%= rank_td rank_single, "continent" %>
             <%= rank_td rank_single, "world" %>
             <td class="single">
-              <%= link_to_ranking rank_single.event_id, :single do %>
+              <%= link_to rankings_path(rank_single.event_id, "single"), class: "plain" do %>
```

Comment:
Great you got path helpers to work =)

Reply:
They work as long as there's an `as: :something` on the route, I couldn't make them work without it :confused: